mercury.spec

```
@@ -62,7 +61,7 @@ Mercury devel
 %autosetup -p1
 %else
 %setup -q -n mercury-%dl_version
-%patch1 -p1
+# %patch1 -p1
```

Maybe it's time for:

```suggestion
%autosetup -n mercury-%dl_version
```

which applies patches if they exist, but doesn't if they don't.  https://rpm.org/user_doc/autosetup.html